chips/lowrisc/README.md

```diff
@@ -0,0 +1,6 @@
+LoWRISC Peripherals
+=======================
+
+This crate contains various peripherals shared between various LowRISC
+cores.
```

Comment:
Since `MachineTimer` already exists, I think we just need GPIO support to merge this.

Reply:
I am working on getting the MachineTimer (and interrupts) working locally. I'll update the PR when I have it working.